test/test_affine.jl

```diff
@@ -417,4 +417,36 @@
             @test all(abs.(p.constraints[1].dual - dual) .<= TOL)
         end
     end
+
+    @testset "Partial transpose" begin
+        dims = [2,3,4]
+        d = prod(dims)
+        A = rand(ComplexF64,2,2)
+        B = rand(ComplexF64,3,3)
+        C = rand(ComplexF64,4,4)
+        M = kron(A,B,C)
+        Mt1 = kron(transpose(A),B,C)
+        Mt2 = kron(A,transpose(B),C)
+        Mt3 = kron(A,B,transpose(C))
+
+        Rt1 = ComplexVariable(d,d)
+        Rt2 = ComplexVariable(d,d)
+        Rt3 = ComplexVariable(d,d)
+        S = rand(ComplexF64,d,d)
+        solve!(satisfy(partialtranspose(Rt1, 1, dims) == S ),SCSSolver(verbose=false))
+        solve!(satisfy(partialtranspose(Rt2, 2, dims) == S ),SCSSolver(verbose=false))
+        solve!(satisfy(partialtranspose(Rt3, 3, dims) == S ),SCSSolver(verbose=false))
```

Comment:
here, it should be `solver` instead of `SCSSolver(verbose=false)`, since this testset is in a loop that will try several different solvers

Reply:
done.